packages/flutter/lib/src/widgets/sliver.dart

```diff
@@ -803,6 +803,60 @@ class SliverGrid extends SliverMultiBoxAdaptorWidget {
        ),
        super(delegate: SliverChildListDelegate(children));
 
+  /// Creates a sliver that places multiple box children in a two dimensional
+  /// arrangement.
+  ///
```

Comment:
On our other slivers we leave a breadcrumb upfront back to `CustomScrollView.slivers`. Let's add that as well here :)

```suggestion
  ///
  /// _To learn more about slivers, see [CustomScrollView.slivers]._
  ///
```